todo/migrations/0004_alter_task_options.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 4.2.7 on 2023-11-08 16:48
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("todo", "0003_alter_task_deadline"),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name="task",
+            options={"ordering": ("is_done", "-created_at")},
+        ),
+    ]
```

Comment:
Add one migration per the whole module within one pull request. It is a good practice to keep the number of migrations as low as possible, besides that each migration in its current form does not have significant changes, so all of them could be implemented at once.